### PR TITLE
Revert Ceph docs edit -- the problem was elsewhere

### PR DESCRIPTION
**Pull request by jistr**
PR https://github.com/openstack-k8s-operators/data-plane-adoption/pull/363 made the Ceph docs slightly inconsistent and didn't seem to fix the docs issue we saw in https://github.com/openstack-k8s-operators/data-plane-adoption/pull/338 -- the problem is likely in that PR.

Let's revert https://github.com/openstack-k8s-operators/data-plane-adoption/pull/363 to make Ceph docs consistent again. I created a separate commit to fix the quoting issue in one of the snippets, i think we want to keep that.

### CODE REVIEW

**Review by fmount — APPROVED** [review]
/lgtm

**Comment by fmount** [comment]
ack and thanks for confirming that syntax was correct.

**Comment by jistr** [comment]
recheck error fetching from rubygems.org

**Comment by jistr** [comment]
@klgill No, the issue was unrelated, it was missing end of the code block `----` in PR 338. Regarding `[source,bash]` i only put the state back into how it was before. I saw at least one other Ceph module has `[source,bash]`, so it seemed random to remove it from that one doc, and i put it back to the original state before #363 was merged. #363 was merged only because we thought the CI was broken, otherwise we'd be looking at that patch more carefully i think.

So in conclusion, nothing of importance happened here i think :grin: . PR 363 landed and here we immediately reverted it after we discovered it wasn't the fix we were looking for.

**Comment by jistr** [comment]
We kept just this bit from PR 363 -- https://github.com/openstack-k8s-operators/data-plane-adoption/pull/367/commits/cbcf2dbf686bd64b72d2190a8e0b3b095cbd0a74
